mslib/mswms/wms.py

```diff
@@ -79,33 +79,40 @@
 realm = 'Mission Support Web Map Service'
 app.config['realm'] = realm
 
+
+class default_mswms_settings(object):
+    base_dir = os.path.abspath(os.path.dirname(__file__))
+    xml_template_location = os.path.join(base_dir, "xml_templates")
+    service_name = "OGC:WMS"
+    service_title = "Mission Support System Web Map Service"
+    service_abstract = ""
+    service_contact_person = ""
+    service_contact_organisation = ""
+    service_contact_position = ""
+    service_address_type = ""
+    service_address = ""
+    service_city = ""
+    service_state_or_province = ""
+    service_post_code = ""
+    service_country = ""
+    service_fees = ""
+    service_email = ""
+    service_access_constraints = "This service is intended for research purposes only."
+    register_horizontal_layers = []
+    register_vertical_layers = []
+    register_linear_layers = []
+    data = {}
+    enable_basic_http_authentication = False
+    __file__ = None
+
+
+mswms_settings = default_mswms_settings()
+
 try:
-    import mswms_settings
+    import mswms_settings as user_settings
+    mswms_settings.__dict__.update(user_settings.__dict__)
```

Comment:
we use the mswms_settings also to patch some parts on our server, how do we do this in the new approach?

Reply:
In either case, due to the nature of import, the patching must be done by "importing" the file first and patching it before wms.py is imported. Then the import here will "simply" be refered to the previously imported&patched file.